cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/fetchdata.curatoroperator.io/customresourcedefinition.yaml

```diff
@@ -3,6 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
```

Comment:
this value is not needed, and will end up failing our upcoming [kubeval](https://github.com/instrumenta/kubeval) ci check

Reply:
Sure. Will remove and push the changes.